timereport/chalicelib/lib/factory.py

```diff
@@ -26,6 +26,9 @@ def factory(order):
         date_obj_stop = datetime.strptime(date_str_stop, format_str)
         for d in date_range(date_obj_start, date_obj_stop):
             dates.append(d)
+    elif "today" in date_str:
```

Comment:
I usually prefer avoiding elif where an ordinary if is also possible

Reply:
updated!